src/components/axes/timeAxis.ts

```diff
@@ -278,15 +289,33 @@ export module Axis {
       return Math.min(this._getIntervalLength(config), this.width()) >= worstWidth;
     }
 
+    protected _getSize(availableWidth: number, availableHeight: number) {
+      // Makes sure that the size it requires is a multiple of tier sizes, such that
+      // we have no leftover tiers
+
+      var size = super._getSize(availableWidth, availableHeight);
+      size.height = this._tierHeights.reduce(function(prevValue, currValue, index, arr) {
+        return (prevValue + currValue > size.height) ? prevValue : (prevValue + currValue);
+      });
+      return size;
+    }
```

Comment:
@jtlan Can you verify if this is the correct usage of _getSize? I know that you added this endpoint recently so you should have the best context here